Screens/NewRace.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from 'react'
+import {
+  Alert,
+  Button,
+  Picker,
+  StyleSheet,
+  Text,
+  TextInput,
+  View
+} from 'react-native'
+
+class NewRace extends Component {
```

Comment:
>NewRace [](start = 6, length = 7)

For Form type components like this - NativeBase has some great themes for getting a cross-platform native look-and-feel. @rachelnicole has some experience with NativeBase.